src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -620,7 +696,7 @@ private String extractUser(HttpServerRequest request) {
         return user;
     }
 
-    private void checkLocked(String queue, final HttpServerRequest request, final Handler<AsyncResult<Boolean>> handler) {
+    private void checkLocked(String queue, final HttpServerRequest request, final Handler<Void> handler) {
```

Comment:
What is this modification good for?

Reply:
Reverted change from preparing PR #63 (See RedisquesHttpRequestHandler.java:623), which is not used now.